docs/tutorials/rfc2136.md

```diff
@@ -12,6 +12,38 @@ key "externaldns-key" {
 ```
 - `Warning!` Bind server configuration should enable for this key AFXR zone transfer protocol. It is used for listing DNS records.
 
+```text
+# cat /etc/named.conf
+...
+include "/etc/rndc.key";
+
+controls {
+inet 123.123.123.123 port 953 allow { 10.x.y.151; } keys { "externaldns-key"; };
```

Comment:
can you indent the config file sections to make it easier to read?

Reply:
For sure! Change added.